src/ceph_fuse.cc

```diff
@@ -143,7 +143,12 @@ int main(int argc, const char **argv, const char *envp[]) {
 	  char buf[5050];
 	  string mountpoint = cfuse->get_mount_point();
 	  snprintf(buf, 5049, "fusermount -u -z %s", mountpoint.c_str());
-	  system(buf);
+	  int umount_r = system(buf);
+	  umount_r = WEXITSTATUS(umount_r);
+	  if (umount_r) {
+	    cerr << "got error " << cpp_strerror(umount_r)
```

Comment:
system(3) 
RETURN VALUE:

"The value returned is -1 on error (e.g., fork(2) failed), and the return status of the command otherwise.  This latter return status is in the format
       specified  in  wait(2).  Thus, the exit code of the command will be WEXITSTATUS(status).  In case /bin/sh could not be executed, the exit status will
       be that of a command that does exit(127)."

Here are the cases:

return -1 (before WEXITSTATUS) fork() failed, errno may be set but this isn't documented on man page
if WIFEXITED(result) is true then you can check WEXITSTATUS(result)
WEXITSTATUS 127 - /bin/sh wouldn't execute
WEXITSTATUS 1 - fusermount command failed
WEXITSTATUS 0 - everything ok
Unlikely but possible that WIFSIGNALED is true because of a segfault or signal sent to process.